SkietbaanBE/SkietbaanBE/Controllers/GroupsController.cs

```diff
@@ -13,6 +13,8 @@ namespace SkietbaanBE.Controllers
     [Route("api/Groups")]
     public class GroupsController : Controller
     {
+        EmailService mailS = new EmailService();
```

Comment:
please rename this variable to something like "mailService"

Reply:
i will take it out i'm no longer using the email